### ISSUE TITLE
Is "pid" example in your docs wrong? 

### ISSUE THREAD

**Issue by davehayes** [jhthorsen/mojo-ioloop-readwritefork#8]
In the example in the "pid" section, you say this will work:

``` perll
  $fork->on(spawn => sub { my $self = shift; warn $self->pid });
  $fork->run("bash", -c => q(echo $YIKES foo bar baz));
```

Don't you really mean this?

``` perll
  $fork->on(fork => sub { my $self = shift; warn $self->pid });
  $fork->run("bash", -c => q(echo $YIKES foo bar baz));
```

**Comment by jhthorsen** [comment]
Yes, you're absolutely right!

Thank you 👍

**Comment by jhthorsen** [comment]
Fixed in 41c595a.

**Comment by davehayes** [comment]
Thank you! :thumbsup:
